utils/doclint/api_parser.js

```diff
@@ -122,7 +122,18 @@ class ApiParser {
     if (!method)
       throw new Error('Invalid method ' + match[2] + '.' + match[3]);
     if (match[1] === 'param') {
-      method.argsArray.push(this.parseProperty(spec));
+      const arg = this.parseProperty(spec);
+      const existingArg = method.argsArray.find(m => m.name === arg.name || m.langs.aliases && Object.values(m.langs.aliases).indexOf(arg.name) !== -1);
+      if (existingArg) {
+        for (const lang of arg.langs.only) {
+          if (existingArg.name !== arg.name && existingArg.langs.aliases[lang] !== arg.name)
```

Comment:
Remove this check, it is to override in anextensible manner without the core knowing.

Reply:
Done.